autowebcompat/utils.py

```diff
@@ -192,7 +192,7 @@ def read_labels(file_name='labels.csv'):
         with open(file_name, 'r') as f:
             next(f)
             reader = csv.reader(f)
-            labels = {row[0]: row[1] for row in reader}
+            labels = {row[0]: row[1] for row in reader if len(row) == 2}
```

Comment:
This check should no longer be needed on latest master (unless you're generating labels on Windows, which sometimes can cause problems. Are you?)

Reply:
I was having this problem on MacOS and Ubuntu, but the problem went away with the check.

Reply:
OK, can you try again now with latest master? It should be fixed

Reply:
@sdv4 ?